src/Action/UserFlairAction.ts

```diff
@@ -28,51 +24,49 @@ export class UserFlairAction extends Action {
     const dryRun = runtimeDryrun || this.dryRun;
     let flairParts = [];
 
-    if (this.flair_template_id !== '') {
+    if (this.flair_template_id !== undefined) {
       flairParts.push(`Flair template ID: ${this.flair_template_id}`)
+      if(this.text !== undefined || this.css !== undefined) {
+        this.logger.warn('Text/CSS properties will be ignored since a flair template is specified');
+      }
     } else {
-      if (this.text !== '') {
+      if (this.text !== undefined) {
         flairParts.push(`Text: ${this.text}`);
       }
-      if (this.css !== '') {
+      if (this.css !== undefined) {
         flairParts.push(`CSS: ${this.css}`);
       }
     }
 
-    const flairSummary = flairParts.length === 0 ? 'No user flair (unflaired)' : flairParts.join(' | ');
+    const flairSummary = flairParts.length === 0 ? 'Unflair user' : flairParts.join(' | ');
     this.logger.verbose(flairSummary);
-    if (item && item?.author) {
-      if (!this.dryRun) {
-        if (this.flair_template_id !== '') {
+
+    if (!this.dryRun) {
```

Comment:
It looked like your use of dryrun was incorrect since the else statement would still run `assignFlair` for the author...i think maybe this was left over from debugging or just overlooked? I believe this is the correct intention

Reply:
Yeah, I've overlooked it. Thanks!